.circleci/config.yml

```diff
@@ -193,6 +193,34 @@ jobs:
           channel: << pipeline.parameters.alerts-slack-channel >>
           template: basic_fail_1
 
+  e2e_environment_test:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.40.0-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E tests
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui.git .
```

Comment:
I considered whether cloning it was still the right approach since this job is already operating from the context of this repo now, and for it to behave more like `unit_test`

I think sticking with the same approach of the old repo is likely the simplest approach to start with as it's similar to what we know works. If it does work, then we can decide if we want to change how it runs to be more like `unit_test`.

Reply:
We don't seem to be doing any [cloning in CAS3](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/blob/main/.circleci/config.yml), so I'm not sure what the result of this would be, but no harm in testing.

Reply:
This has made me think that I could try to add another job in for the non-cloning path and see if it works!

Reply:
[It worked without cloning quite easily](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-community-accommodation-tier-2-ui/3208/workflows/d3548fad-7978-46ce-8b7e-584c3106c192/jobs/12782). I worked out how to test it before pushing too. I'm going to reforge this PR to basically swap out the insides of the same job.